x-pack/packages/kbn-elastic-assistant/impl/assistant/chat_send/use_chat_send.tsx

```diff
@@ -112,6 +115,24 @@ export const useChatSend = ({
     ]
   );
 
+  const handleRegenerateResponse = useCallback(async () => {
+    const updatedMessages = removeLastMessage(currentConversation.id);
+    const rawResponse = await sendMessages({
```

Comment:
I don't think it's related to the regenerate response action, but I am sometimes seeing the scroll to bottom behavior not working. Can't repro 100% yet, but will add steps once I can (nearly positive I didn't touch the mouse/keyboard, so shouldn't be an interrupt here).


<p align="center">
  <img width="700" src="https://github.com/elastic/kibana/assets/2946766/9a8f3592-f04f-4524-9707-bc720409ae39" />
</p>